packages/customer-account-ui-extensions/src/extension-points/standard-api/navigation.ts

```diff
@@ -0,0 +1,30 @@
+export type NavigationType = 'push' | 'replace' | 'traverse';
+
+export interface NavigationOptions {
+  state?: Record<string, any>;
+  type: 'push' | 'replace';
```

Comment:
shouldn't that be `history`?  https://developer.mozilla.org/en-US/docs/Web/API/Navigation/navigate#:~:text=as%20info.-,history,-An%20enumerated%20value

https://github.com/Shopify/ui-api-design/discussions/21#discussioncomment-6071362